backend/e2etests/ratings.test.js

```diff
@@ -56,7 +56,7 @@ describe(`${endpoint}`, function () {
         .then((res) => {
           const body = JSON.stringify(res.body)
           expect(body).contains('"hits":');
-          expect(body).contains('"limit":20,"nbHits":1,"offset":0}');
+          expect(body).contains('"limit":20,"nbHits":0,"offset":0}');
```

Comment:
This `expect` case here does not match with the test label. We should find values for `company` and `jobtitle` that return something.

Reply:
i added more entries in the salaries table fixture to satisfy this because of the maxEntryBeforeDisplay rules when the company filter is filled